cnd/src/http_api/markets/get_btc_dai.rs

```diff
@@ -20,11 +20,20 @@ pub fn route(facade: Facade) -> impl Filter<Extract = impl Reply, Error = Reject
         })
 }
 
+/// Retrieves "executable" orders: orders that have expiries that match the safe
+/// expiries determined by the expiries module.
 async fn handler(facade: Facade) -> Result<impl Reply> {
     let mut orders = siren::Entity::default();
     let local_peer_id = facade.swarm.local_peer_id();
 
-    for (maker, order) in facade.swarm.btc_dai_market().await {
+    let executable_orders = facade
+        .swarm
+        .btc_dai_market()
+        .await
+        .into_iter()
+        .filter(|(_, order)| has_executable_expiries(order));
```

Comment:
```suggestion
        .filter(|(_, order)| has_viable_expiries(order));
```
I think you replaced the original `useful` terminology from the expiry module by `executable`. After a lot of back and forth, @tcharding and I settled on `viable` as the best descriptive for those expiries.

Reply:
fixed